doc/src/build/cli-client.md

```diff
@@ -507,25 +507,24 @@ id: 0x471c8e241d0473c34753461529b70f9c4ed3151b[1]
 
 ## Publish packages
 
-In order for user-written code to be available in Sui, it must be
-*published* to Sui's [distributed ledger](../learn/how-sui-works.md#architecture).
-Please see the [Move developer documentation](move/index.md) for a
-description on how to [write a simple Move code package](move/write-package.md),
-which we can publish using Sui client's `publish` command.
+You must publish packages to the Sui [distributed ledger](../learn/how-sui-works.md#architecture) for the code you developed to be available in Sui. To publish packages with the Sui client, use the `publish` command.
+
+The publish command requires that you specify the directory where your package lives using the `--path` parameter. The value is the path to the `my_move_package` as per the [package creation description](move/write-package.md). You must also provide a `gas` object to pay for publishing the package and a `gas-budget` value.
 
-> **Important:** All calls to functions in the `debug` module must be removed from no-test code
-> before the new module can be published (test code is marked with the `#[test]` annotation).
+Parameters used for the `publish` command example:
+* `--path` - Defines the path to the Move package to publish.
+* `--gas` - The Coin object used to pay for gas.
+* `--gas-budget` - Gas budget for running module initializers. 
+* `--verify-dependencies` - Optional flag to have the CLI check that a dependency exists on-chain. 
```

Comment:
Suggestion to indicate that it checks all dependencies, and beyond just existing, they need to match (please feel free to rephrase).

```suggestion
* `--verify-dependencies` - Optional flag to have the CLI check that all dependencies match their on-chain counterparts. 
```